components/brave_wallet_ui/common/hooks/explorer.ts

```diff
@@ -10,12 +10,18 @@ import { BraveWallet, BlockExplorerUrlTypes } from '../../constants/types'
 import Amount from '../../utils/amount'
 
 export function buildExplorerUrl (
-    network: BraveWallet.NetworkInfo, type: BlockExplorerUrlTypes,
-    value?: string, id?: string) {
+  network: BraveWallet.NetworkInfo, type: BlockExplorerUrlTypes,
+  value?: string, id?: string) {
   const explorerURL = network.blockExplorerUrls[0]
 
+  const fallbackURL = `${explorerURL}/${value}}`
```

Comment:
Is this value ever used? Looks like it has had a bug for a while (extra `}` )

Reply:
Good catch! Removed the extra bracket!